src/components/CoboVault/styles.css

```diff
@@ -0,0 +1,42 @@
+.qr-player__container {
```

Comment:
This is another one that's on us for having old standards mixed with new ones, but prefer sass modules for styles. See `ColdcardFileReader.module.scss` for the most recent example.

Reply:
I don't have a particularly strong preference myself, but sass would allow for hierarchal declaration of styles. 

`.qr-player { .title {} }` instead of `.qr-player__title`. I'm fine with this format too but worth mentioning since sass is included in the build process anyway.